components/more-less/more-less.js

```diff
@@ -383,6 +314,23 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 		this.dispatchEvent(new CustomEvent(e.type, { bubbles: true, composed: true, detail: e.detail }));
 	}
 
+	#handleSlotChange(e) {
+		this.__reactToChanges();
+		this.__resizeObserver.disconnect();
+		this.__mutationObserver.disconnect();
+
+		const children = e.target.assignedNodes({ flatten: true }).filter((node) => node.nodeType === Node.ELEMENT_NODE);
```

Comment:
Worth noting that this is going to filter out text nodes - I suspect it's not possible to observe them. So between that and not observing the slot element itself, we wouldn't detect the addition of text nodes as direct children of the slot.

Reply:
Aside from that, we're safe to use `assignedElements`, which avoids us having to filter them ourselves.